dcicutils/glacier_utils.py

```diff
@@ -0,0 +1,321 @@
+import boto3
+from typing import Union, List
+from concurrent.futures import ThreadPoolExecutor
+from tqdm import tqdm
+from .misc_utils import PRINT
+from .ff_utils import get_metadata, get_health_page, patch_metadata
+from .creds_utils import KeyManager
+
+
+GLACIER_CLASSES = [
+    'GLACIER',
+    'DEEP_ARCHIVE'
+]
```

Comment:
I'm guessing `GLACIER` here refers to what is now Glacier Flexible Retrieval per documentation here: https://aws.amazon.com/s3/storage-classes/. I don't know if that value still corresponds to Glacier Flexible Retrieval, or covers both Glacier Flexible Retrieval and Glacier Instant Retrieval now. If you're confident on the topic, then consider adding a quick comment for clarification. If not, consider reducing to only `DEEP_ARCHIVE` (since we've only ever used Glacier Deep Archive as far as I know), checking all current Glacier storage classes are covered with these 2 options, or adding a comment to look more into this later.